exercises/proverb/src/main/java/Proverb.java

```diff
@@ -0,0 +1,12 @@
+class Proverb {
+    private String[] words;
+
+    Proverb(String[] words) {
+        this.words = words;
```

Comment:
This should be replaced with `throw new UnsupportedOperationException("Delete this statement and write your own implementation.");`. We only want to provide the users with the signatures for the exercise, not any of the implementation :)